docs/conf/modules.conf.example

```diff
@@ -464,6 +464,11 @@
 # This module is oper-only.
 # To use, CHGNAME must be in one of your oper class blocks.
 #<module name="chgname">
+#
+#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#-#
+# Connection class ban module: Adds support for extban 'n' which 
+# matches against the class name of the user's connection.
```

Comment:
I think we should mention that for this module to work properly on a network with multiple servers, all servers on the network should refer to the same class with the same name. For example, if there is a webirc class on one server and not on another, it won't work.